admin-portal/src/pages/editPledge/components/pledgeSummary/pledgeSummary.tsx

```diff
@@ -1,44 +1,20 @@
-import React, { ChangeEvent } from "react";
+import React from "react";
 import "./pledgeSummary.css";
 import { Pledge } from "../../../../types/Pledge";
 import { AidPackage } from "../../../../types/AidPackage";
+import PledgeStatusSelector from "../pledgeStatusSelector/pledgeStatusSelector";
 
 interface PledgeSummaryProps {
   pledge: Pledge;
-  onStatusChange: (status: Pledge.Status) => void;
   aidPackageStatus: AidPackage.Status;
+  setPledge: (prevPledge: any) => any;
```

Comment:
Isn't this a `Pledge` type?

Reply:
nope, it has some weird way of using setState. It's giving type _any_ where it's used.
That was a previous implementation I think.

Reply:
![image](https://user-images.githubusercontent.com/47103883/187963747-cb84f49d-5a8e-4f24-b3f7-fb8984ede11e.png)
over here, type _any_ has to be used, eslint shows an error otherwise

Reply:
![image](https://user-images.githubusercontent.com/47103883/187964533-0b24a8cc-f6bc-4571-ae91-f3c44e32279c.png)

Reply:
oh okay, we can get rid of `any` types when we are doing refactoring.